enarx-mirrorlabels

```diff
@@ -0,0 +1,173 @@
+        'color': 'f22933',
+        'old_names': {},
+    },    
+    'ibm pef': {
+        'description': 'Issues related to IBM PEF',
+        'color': '0043ce',
+        'old_names': {},
+    },    
+    'infrastructure': {
+        'description': 'Improvements or additions to project infrastructure',
+        'color': 'ffdda8',
+        'old_names': {},
+    },    
+    'intel sgx': {
+        'description': 'Issues related to Intel SGX',
+        'color': '0071c5',
+        'old_names': {},
+    },    
+    'invalid': {
+        'description': "This doesn't seem right",
+        'color': 'e4e669',
+        'old_names': {},
+    },    
+    'mentorship': {
+        'description': 'A request for mentorship on the project.',
+        'color': '0e8a16',
+        'old_names': {},
+    },    
+    'meta': {
+        'description': 'Larger project tasks and goals',
+        'color': '149e7b',
+        'old_names': {},
+    },    
+    'question': {
+        'description': 'Further information is requested',
+        'color': 'd876e3',
+        'old_names': {},
+    },    
+    'research': {
+        'description': '',
+        'color': 'ed8661',
+        'old_names': {},
+    },    
+    'security': {
+        'description': 'Issues that have security implications',
+        'color': 'ff0000',
+        'old_names': {},
+    },    
+    'syscall': {
+        'description': 'syscall-related issues and PRs',
+        'color': 'a51f3c',
+        'old_names': {},
+    },    
+    'wasm': {
+        'description': 'Issues related to WebAssembly',
+        'color': '654EF0',
+        'old_names': {},
+    },    
+    'wontfix': {
+        'description': 'This will not be worked on',
+        'color': 'ffffff',
+        'old_names': {},
+    }
+}
+
+# Construct a set of key names for use later.
+organization_label_names = {l for l in ORGANIZATION_LABELS.keys()}
+
+# Construct a dictionary of old label names and their new names to be used in renaming.
+all_old_names = {}
+for k in organization_label_names:
+    for old_name in ORGANIZATION_LABELS[k]['old_names']:
+        all_old_names[old_name] = k
+
+github = enarxbot.connect()
+org = github.get_organization('enarx')
+
+for repo in org.get_repos():
+    repo_labels = {l.name: l for l in repo.get_labels()}
+    repo_label_names = {l for l in repo_labels.keys()}
+
+    # Find the set of labels on the repo that need to be renamed.
+    to_be_renamed = repo_label_names & all_old_names.keys()
+
+    # Rename, if there are labels to be renamed.
+    if len(to_be_renamed) > 0:
+        for name in to_be_renamed:
+            print(f"{repo.name}: renaming label {name} to {all_old_names[name]}")
+            label = repo_labels[name]
+            label.edit(name=all_old_names[name], description=label.description, color=label.color)
+        # To prevent actions being taken on stale cache entries, don't do
+        # further processing after renames happen (if they happen).
+        continue
```

Comment:
```suggestion
        for name in to_be_renamed:
            print(f"{repo.name}: renaming label {name} to {all_old_names[name]}")
            label = repo_labels[name]
            label.edit(name=all_old_names[name], description=label.description, color=label.color)

        # To prevent actions being taken on stale cache entries, don't do
        # further processing after renames happen (if they happen).
        continue
```